packages/app/src/containers/SwatchesInspector.tsx

```diff
@@ -65,16 +37,37 @@ export default memo(function ComponentsInspectors() {
     [dispatch, selectedSwatches],
   );
 
+  const handleChangeOpacity = useCallback(
+    (value) => dispatch('setSwatchOpacity', ids, value),
+    [dispatch, ids],
+  );
+  const handleNudgeOpacity = useCallback(
+    (value) => dispatch('setSwatchOpacity', ids, value, 'adjust'),
+    [dispatch, ids],
+  );
+  const handleChangeColor = useCallback(
+    (value) => dispatch('setSwatchColor', ids, value),
+    [dispatch, ids],
+  );
+
   if (selectedSwatches.length === 0) return null;
 
   return (
-    <Fragment key="layout">
+    <>
       <NameInspector
         names={selectedSwatches.map((v) => v.name)}
         onNameChange={handleNameChange}
       />
-      <ColorPickerInspector />
-      <Spacer.Vertical size={10} />
-    </Fragment>
+      <Divider />
+      <Container>
+        <ColorInspector
```

Comment:
Oh I like it more like this, it also follows what I did before with the name! Make sense